StarDb/AgMLGeometry/CreateGeometry.h

```diff
@@ -2,8 +2,6 @@
 
 #if !defined(__CINT__) || defined(__CLING__)
 #include "StBFChain/StBFChain.h"
-#include "StarVMC/StarAgmlLib/AgModule.h"
```

Comment:
Probable culprit for the test failures.  The AgML library codes are overly complex... especially the shape and structure handling codes.  Would not be surprise if cling doesn't handle them well...